desc/plotting.py

```diff
@@ -1282,6 +1282,132 @@ def plot_surfaces(eq, rho=8, theta=8, zeta=None, ax=None, **kwargs):
     return fig, ax
 
 
+def plot_boundary(eq, zeta=None, axis=True, ax=None, **kwargs):
+    """Plot stellarator boundary at multiple toroidal coordinates.
+
+    Parameters
+    ----------
+    eq : Equilibrium
+        Object from which to plot.
+    zeta : int or array-like or None
+        Values of zeta to plot boundary surface at.
+        If an integer, plot that many contours linearly spaced in [0,2pi).
+        Default is 1 contour for axisymmetric equilibria or 4 for non-axisymmetry.
+    axis : bool
```

Comment:
might want to rename this to avoid confusion with `ax` argument.

Reply:
(also not sure its totally necessary, I haven't seen the axis position included in these sorts of plots before)

Reply:
Renamed to `plot_axis`. 

I have seen it before and I like it, but I changed the default to not include the axis points.